Library/Homebrew/cmd/test-bot.rb

```diff
@@ -520,7 +521,7 @@ def formula formula_name
               test "brew", "uninstall", "--force", *unchanged_build_dependencies
               unchanged_dependencies -= unchanged_build_dependencies
             end
-            test "brew", "install", bottle_filename
+            test "brew", "install", "--force-bottle", bottle_filename
```

Comment:
I'm still not familiar with the test bot code, but can this be fixed in `formula_installer`?
```diff
 Library/Homebrew/formula_installer.rb | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/Library/Homebrew/formula_installer.rb b/Library/Homebrew/formula_installer.rb
index a6264ee..a4787a0 100644
--- a/Library/Homebrew/formula_installer.rb
+++ b/Library/Homebrew/formula_installer.rb
@@ -61,11 +61,10 @@ class FormulaInstaller
     return false if @pour_failed
 
     bottle = formula.bottle
-    return true  if force_bottle? && bottle
+    return true  if (force_bottle? && bottle) || formula.local_bottle_path
     return false if build_from_source? || build_bottle? || interactive? || formula.file_modified?
     return false unless options.empty?
 
-    return true if formula.local_bottle_path
     return false unless bottle && formula.pour_bottle?
 
     unless bottle.compatible_cellar?
```